xbmc/cores/VideoRenderers/RenderManager.cpp

```diff
@@ -816,7 +816,7 @@ void CXBMCRenderManager::Render(bool clear, DWORD flags, DWORD alpha, bool gui)
 
 bool CXBMCRenderManager::IsGuiLayer()
 {
-  { CSingleLock lock(m_presentlock);
+  { CSharedLock lock(m_sharedSection);
```

Comment:
it is included in the header https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoRenderers/RenderManager.h#L28